pkgs/development/compilers/llvm/3.9/llvm.nix

```diff
@@ -88,6 +97,9 @@ in stdenv.mkDerivation rec {
     ++ stdenv.lib.optionals (isDarwin) [
     "-DLLVM_ENABLE_LIBCXX=ON"
     "-DCAN_TARGET_i386=false"
+  ] ++ stdenv.lib.optionals (buildPlatform != hostPlatform) [
+    "-DCMAKE_CROSSCOMPILING=True"
```

Comment:
Looks like it https://cmake.org/cmake/help/v3.0/variable/CMAKE_CROSSCOMPILING.html